### PR TITLE
Destination details

### PR DESCRIPTION
**Pull request by techeduk8r**
Detail destination pages created (2 examples). Still need button styling, but otherwise, pages are complete.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 *.psd
 popular-destinations.html
 slideshow.css
-
+dest-details.css
```
Comment (loganmorganml1) [inline .gitignore:4]: I see you've added the css for this in the main styles file, but this is linked to the seperate css file you've created. Do you want to link to the `style.css` before merging?

**Review by loganmorganml1 — CHANGES_REQUESTED** [review]
More of a question than request, see comment attached.

File: .gitignore (hunk 1 above)
Comment (techeduk8r) [reply .gitignore:4]: Thanks for catching that! This file was after I lost all of the destination code and I forgot that small but important piece. It's all updated.

**Review by loganmorganml1 — APPROVED** [review]
Looks good, approved!

**Pull request merged**